bolt/src/main/java/com/slack/api/bolt/App.java

```diff
@@ -930,16 +930,16 @@ protected Response runHandler(Request slackRequest) throws IOException, SlackApi
                 case Event: {
                     if (eventsDispatcher.isRunning()) {
                         eventsDispatcher.enqueue(slackRequest.getRequestBodyAsString());
+                        return Response.ok();
```

Comment:
This is a primitive event dispatcher in slack-app-backend module. Although most developers probably don't use this in Bolt for Java, Bolt enables using it in Bolt apps. When developers use this, event requests should be still acknowledged anyway. So, I added the return statement here.